doc/source/dev/toolchain.rst

```diff
@@ -72,20 +73,24 @@ access to a few functions.  Use in conjunction with ``<numpy/npy_math.h>``.
  mid-block variable declarations   Declare variables at the top of block
 ================================  ========================================================
 
+For C++, we currently suggest that anything beyond C++11 is going to be impossible
+for a very long time because of ecosystem support restrictions.
+
 Fortran Compilers
 -----------------
 
-[*What is the requirement on a Fortran compiler*]
+Generally, any well-maintained compiler is likely suitable. gfortran >= 4.8.0,
+ifort, or flang can be used to build SciPy.
```

Comment:
"generally ... likely" - would be good to be more exact. state what we know and don't know. Something like:

SciPy contains only Fortran 77 and Fortran 90 code, therefore most compilers that are maintained are fine. Known to work:
- gfortran >= 4.8.0
- Intel Fortran
- Flang (not on Windows yet)